src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -106,14 +107,15 @@ impl Renderer for HtmlHandlebars {
 
                         // Write to file
                         info!("[*] Creating {:?} ✓", filename.display());
-                        try!(book.write_file(filename, &rendered.into_bytes()));
+                        book.write_file(filename, &rendered.into_bytes())?;
 
                         // Create an index.html from the first element in SUMMARY.md
                         if index {
                             debug!("[*]: index.html");
 
                             let mut content = String::new();
-                            let _source = try!(File::open(book.get_dest().join(&ch.path.with_extension("html"))))
+                            let _source = File::open(book.get_dest().join(&ch.path.with_extension("html")))
+                                ?
```

Comment:
The questionmark should definitely not go on its own line here. This is probably a bug in rustfmt, but it can certainly be solved on our side by moving `book.get_dest()...` into a temporary variable.